spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/config/DataFlowServerProperties.java

```diff
@@ -32,6 +33,10 @@
 
 	static final String PREFIX = "deployer";
 
+	static final String AETHER_PROXY_PREFIX = "aether.proxy.";
```

Comment:
as discussed in a call, maybe use `maven.proxy` (as aether can be seen as an impl detail).

Or maybe going even further, have those be very generic `http.proxy` settings (if e.g. targeting s3 in the future)